### ISSUE TITLE
Incorrect detection of get-task-allow

### ISSUE THREAD

**Issue by david-antiteum** [akeru-inc/xcnotary#8]
The entitlements file can contain an entry for get-task-allow with a false value. Example:

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
  <dict>
    <key>com.apple.security.get-task-allow</key><false/>
  </dict>
</plist>
```

xcnotary reports incorrectly this entry as a problem.

Thanks

**Comment by davidvartan** [comment]
Thanks for the report! Will commit a fix this week.

**Comment by davidvartan** [comment]
Fixed in v0.4.2
